<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-11T13:55:19 IST" name="Default suite" finished-at="2024-04-11T13:55:27 IST" duration-ms="8323">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:stepdefinitions1.Testrunner1@4a883b15]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-04-11T13:55:19 IST" name="Default test" finished-at="2024-04-11T13:55:27 IST" duration-ms="8323">
      <class name="stepdefinitions1.Testrunner1">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:stepdefinitions1.Testrunner1@4a883b15]" started-at="2024-04-11T13:55:19 IST" name="setUpClass" finished-at="2024-04-11T13:55:19 IST" duration-ms="384" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@58a120b0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:stepdefinitions1.Testrunner1@4a883b15]" started-at="2024-04-11T13:55:19 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-11T13:55:27 IST" duration-ms="7868" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["check login is successful with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["login functionality"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[valid login expected [https://www.facebook.com] but found [https://www.facebook.com/login/?privacy_mutation_token=eyJ0eXBlIjowLCJjcmVhdGlvbl90aW1lIjoxNzEyODIzOTIzLCJjYWxsc2l0ZV9pZCI6MzgxMjI5MDc5NTc1OTQ2fQ%3D%3D]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: valid login expected [https://www.facebook.com] but found [https://www.facebook.com/login/?privacy_mutation_token=eyJ0eXBlIjowLCJjcmVhdGlvbl90aW1lIjoxNzEyODIzOTIzLCJjYWxsc2l0ZV9pZCI6MzgxMjI5MDc5NTc1OTQ2fQ%3D%3D]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at stepdefinitions1.Loginsteps.user_is_naviagted_to_the_home_page(Loginsteps.java:44)
at âœ½.user is naviagted to the home page(file:///C:/Users/Admin/eclipse-workspace/Cucumber/src/test/resources/Features/login1.feature:8)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:stepdefinitions1.Testrunner1@4a883b15]" started-at="2024-04-11T13:55:27 IST" name="tearDownClass" finished-at="2024-04-11T13:55:27 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- stepdefinitions1.Testrunner1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
